### PR TITLE
TinyMCE Button: PHP Tests 

### LINKED ISSUE #6702: TinyMCE Button: PHP Tests
## Task Description

The new TinyMCE button class doesn't have adequate PHP test coverage.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Add some basic PHP Unit tests

## Relevant Technical Choices

There are not lots of unit tests that add value here. But added them anyway. 

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6702

### CODE REVIEW

File: includes/TinyMCE.php [hunk 1]
```diff
@@ -104,9 +111,9 @@ public function enqueue_assets() {
 		$asset = $this->get_asset_metadata( 'tinymce-button' );
 
 		// Only works when wp_register_script and wp_enqueue_script are called separately.
-		wp_register_script( 'web-stories-tinymce-button', false, $asset['dependencies'], $asset['version'], true );
-		wp_enqueue_script( 'web-stories-tinymce-button' );
-		wp_add_inline_script( 'web-stories-tinymce-button', 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
+		wp_register_script( self::SCRIPT_HANDLE, false, $asset['dependencies'], $asset['version'], true );
+		wp_enqueue_script( self::SCRIPT_HANDLE );
+		wp_add_inline_script( self::SCRIPT_HANDLE, 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
```
Comment (swissspidy) [inline includes/TinyMCE.php]: Could we actually use `wp_localize_script` now that we're doing `wp_register_script`?

**Review by swissspidy — APPROVED** [review]
LGTM with one question

File: includes/TinyMCE.php (hunk 1 above)
Comment (spacedmonkey) [reply includes/TinyMCE.php]: Changed.

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
@spacedmonkey FYI, heads up, I am partially reverting the `$this->enqueue_script()` change because it leads to "undefined index" notices inc core.

**Pull request merged**